scripts/bootstrap-ironic.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Shell Opts ----------------------------------------------------------------
+
+set -e -u -x
+set -o pipefail
+
+## Functions -----------------------------------------------------------------
+
+export BASE_DIR=${BASE_DIR:-"/opt/rpc-openstack"}
+source ${BASE_DIR}/scripts/functions.sh
+
+## Main ----------------------------------------------------------------------
+
+# Check the openstack-ansible submodule status
+check_submodule_status
+
+# begin the bootstrap process
+pushd ${OA_DIR}
```

Comment:
pre-config should be done before osa is deployed, this will cause ironic to be initially deployed, but without the network uuid in it's config.
after osa is deployed the 'post-install' piece should be run, this will populate the network uuid in the config
then the os-ironic-install play should be rerun (initially ran in the osa install), this will put the network uuid in the running config